.changeset/silver-plums-speak.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-techdocs': minor
```

Comment:
Since this PR only appears to be affecting exported (but `@internal`-marked) functions, feel free to set this to `patch` instead of `minor`.

While you're updating this!  If you rename to `techdocs-silver-plums-speak.md`, we can expedite getting this merged in.

Reply:
It's done :)